src/app/swiper/keyboard/keyboard-swiper.tab.ts

```
@@ -0,0 +1,31 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ViewEncapsulation,
+} from '@angular/core';
+import {JsonPipe} from '@angular/common';
+import {SwiperComponent, SwiperItemDirective} from 'ng-swiper';
+import {SwiperOptions} from 'swiper/types';
+import {Keyboard} from 'swiper/modules';
+
+@Component({
+    selector: 'app-keyboard-swiper',
```

ESLint

[error] The selector should start with one of these prefixes: "ng" (https://angular.dev/style-guide#style-02-07)

(@angular-eslint/component-selector)